ipapython/ipa_log_manager.py

```diff
@@ -197,6 +250,12 @@ def convert_log_level(value):
     return level
 
 
+# Add info_cmd to logger and root logger
+if not hasattr(logging.getLoggerClass(), "info_cmd"):
+    logging.getLoggerClass().info_cmd = log_info_cmd
+if not hasattr(logging, "info_cmd"):
+    logging.info_cmd = log_info_cmd_root
```

Comment:
This should not be necessary.

Reply:
This is used any more with the latest version.